app/containers/Network/containers/AccessPointDetails/index.js

```diff
@@ -191,7 +177,10 @@ const AccessPointDetails = ({ locations }) => {
   const [updateEquipmentFirmware] = useMutation(UPDATE_EQUIPMENT_FIRMWARE);
 
   const { data: dataFirmware, error: errorFirmware, loading: landingFirmware } = useQuery(
-    GET_ALL_FIRMWARE
+    GET_ALL_FIRMWARE,
+    {
+      variables: { modelId: data?.getEquipment?.model },
```

Comment:
did you test this? it won't work cause it's dependant on the previous useQuery to resolve first and it's not synchronous.